### ISSUE TITLE
Validation loss not converging when finetuning om mp20

### ISSUE THREAD

**Issue by elisathr** [microsoft/mattergen#94]
I'm finetuning the model on the `dft_mag_density` property using the mp_20 dataset. My issue is that the validation loss does not converge, even though the training loss decreases. 

**Training configuration:**
- Batch size: 128 (global batch size, without gradient accumulation)
- Learning rate: 6×10−5 (same as stated in D1.2 in the supplementary)

I would expect both training and validation loss to decrease, but the validation loss does not decrease, even after multiple epochs. I have also tried to finetune on `dft_bulk_modulus` and I get the same behaviour. Here are the loss curves: 

![Image](https://github.com/user-attachments/assets/cd90d0a0-b41c-46ed-8fa0-8923c76063df)
![Image](https://github.com/user-attachments/assets/ea4ed2e3-6252-4a44-bc84-335937e732a0)

I stopped the training after 409 epochs, since it did not improve. Here is the log;

[daily-elevator-2-log.txt](https://github.com/user-attachments/files/19224278/daily-elevator-2-log.txt)

**Comment by txie-93** [comment]
Thanks for your question! Your observation on the validation loss is consistent with our experiments. We've found that the validation loss doesn't change much for fine-tuning. However, if you evaluate the downstream performance, you will see the performance continue to improve as you keep fine-tuning. We recommend to fine-tune 100-500 epochs based on your dataset, and load the last checkpoint to evaluate downstream performance.
